avaframe/com1DFAPy/com1DFA.py

```diff
@@ -1482,6 +1547,44 @@ def plotContours(fig, ax, particles, dem, data, Cmap, unit, last=False):
     return fig, ax, cmap, lev
 
 
+def releaseSecRelArea(cfg, particles, fields, dem):
+    flowDepthField = fields['FD']
+    secondaryReleaseInfo = particles['secondaryReleaseInfo']
+    if secondaryReleaseInfo:
+        secRelRasterList = secondaryReleaseInfo['rasterList']
+        secRelRasterNameList = secondaryReleaseInfo['Name']
+        count = 0
+        for secRelRaster, secRelRasterName in zip(secRelRasterList, secRelRasterNameList):
+            # do the two arrays intersect (meaning a flowing particle entered the
+            # secondary release area)
+            mask = (secRelRaster > 0) & (flowDepthField > 0)
+            if mask.any():
+                # create secondary release area particles
+                log.info('Initializing secondary release area feature %s' % secRelRasterName)
+                secRelParticles, fields = initializeParticles(cfg, secRelRaster, dem)
+
+                # # save particles and fields for the initial time step to file
+                # outDirData = os.path.join(outDir, 'secondaryReleaseArea', secRelRasterName)
+                # exportFields(cfg, [0.], [fields], demOri, outDirData, secRelRasterName)
+                # if 'particles' in cfg['resType']:
+                #     # export particles dictionaries of saving time steps
+                #     savePartToPickle([secRelParticles], outDirData)
```

Comment:
still needed?

Reply:
Depends on if we want to save the secRelease particles initial state